packages/x-charts/src/ChartsLabel/ChartsLabelMark.tsx

```diff
@@ -3,10 +3,15 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import { styled, SxProps, Theme } from '@mui/material/styles';
 import clsx from 'clsx';
+import { SeriesId } from '../models/seriesType/common';
+import { consumeSlots } from '../internals/consumeSlots';
 import { ChartsLabelMarkClasses, labelMarkClasses, useUtilityClasses } from './labelMarkClasses';
-import { consumeThemeProps } from '../internals/consumeThemeProps';
 
 export interface ChartsLabelMarkProps {
+  /**
+   * ID of the series this mark refers to.
+   */
+  seriesId?: SeriesId;
```

Comment:
Probably needs `dataIndex` as well, since for `PieCharts` the entries are based on that 😢

Reply:
Good point! I'll create a demo for pie charts, then 👍

Reply:
Glad I tried to create a demo. It turns out the API didn't allow for the same customization in pie charts as it did for others 😄